.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: required
 language: go
 go:
-- 1.9
+- 1.9.2
```

Comment:
why do we need a superior version?

Reply:
Its a revision; no new features, but fixes, including security ones. See: https://golang.org/doc/devel/release.html#go1.9.minor